### DISCUSSION TITLE
Stand Up Wednesday April 19th

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [Bugs-io/TC3001B#62] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 
**What are your priorities for the next 24 hours?**:
 - 
**Blockers**:
 -
**Shoutouts**:
 -@username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
- pair programming with Brandon to setup pylint action for PRs

**What are your priorities for the next 24 hours?**:
- Login/Signups flow

**Blockers**:
 - None

**Shoutouts**:
 - Brandon for his work with pylint

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Finished Sign Up form design
 - Started form control and handling user data
 - Questioned about Carbon Design and Typescript, as well as NextJS 12 vs 13

**What are your priorities for the next 24 hours?**:
 - Continue working with form control and handling user data.
 
**Blockers**:
 - Second thoughts on Carbon Design and NextJS architecture.

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
- Work with @CarlosCRG19 on pylint action to analize backend python code on every PR
-  Work on automating pylint analysis on local

**What are your priorities for the next 24 hours?**:
- Fix develop branch 

**Blockers**:
- None

**Shoutouts**:
 - @CarlosCRG19 for working so hard

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
 - Continued working on cleaning script API

**What are your priorities for the next 24 hours?**:
 - Cleaning script API

**Blockers**:
 - Google cloud storage
